blogging/blog_guidelines.md

```diff
@@ -0,0 +1,127 @@
+   - appropriate links and tags
+ - While editing the story, click “…”
+ - Click “Add to Publication”
+ - If applicable, uncheck “Allow curators to recommend my story to interested readers.”
+ - Click “Submit to Publication”
+ - The blog will remain Pending until the publication owner posts it
+
+ ## Content guidelines
+
+ As a blog writer you can write about any topic that you're interested in and that will provide value to the Zowe community.  The content on our blog consists of:
+
+ - Articles about core Zowe projects, Zowe incubation projects, or Zowe conformant offerings
+ - Technical content and how-to's
+ - Stories about Zowe deployments
+ - Use cases and success stories
+ - Reports from Zowe events
+
+ Other topics are welcome, but it needs to have a direct link to the Zowe community
+ 
+ Blogs fall into one of three categories and are differentiated to avoid a user visiting a Zowe blogsite from misunderstanding the relationship between whether the content being described is free open source software included with a Long Term Support (LTS) Zowe release, a commermcial vendor Zowe offerings, or a project in its incubation phase.
+
+ - [Zowe Core](#zowe-core)
+ - [Zowe Ecosystem](#zowe-ecosystem)
+ - [Zowe Incubation](#zowe-incubation)
+
+ 
+### Zowe Core
+ 
+ These articles should reference content that is owned and managed by the Open Mainframe Project residing as source code in [https://github.com/zowe](https://github.com/zowe) repositories.  Core Zowe content is available for a user to download on [https://www.zowe.org/#download](https://www.zowe.org/#download) as well as the Visual Studio Marketplace plugin [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe).
+
+ Core Zowe articles may also reference other open source technologies such as [Visual Studio Code](https://code.visualstudio.com/), [Jenkins](https://www.jenkins.io/), and other products available under permissive open source licenses such as [Massachusetts Institute of Technology (MIT)](https://opensource.org/licenses/MIT), [Berkley Software Distribution (BSD)](http://www.ipglossary.com/glossary/berkeley-software-distribution-license-bsd-license/#.XEiHtlwzaUk),  [Eclipse Public License (EPL)](https://www.eclipse.org/legal/epl-2.0/faq.php), [Apache](https://www.apache.org/licenses/), [Microsoft Public License](https://resources.whitesourcesoftware.com/blog-whitesource/top-10-microsoft-public-license-ms-pl-questions-answered), [Common Development and Distribution License(CDDL)](https://resources.whitesourcesoftware.com/blog-whitesource/top-10-common-development-and-distribution-license-cddl-questions-answered).
+
+ A user visiting a blog post on Core Zowe should be able to operate all of the software described in the article through open source license.
```

Comment:
This sounds awkward - and feels to be a duplicate of the previous paragraph. But maybe you had something else in mind ;-)